### PR TITLE
Rewrite guide

### PR DESCRIPTION
**Pull request by fhemberger**
URL for the debugging guide changed to match the other guides.

### CODE REVIEW

File: setup/www/resources/config/nodejs.org [hunk 1]
```diff
@@ -426,4 +426,6 @@ server {
 
     # Rename 'Stable' release to 'Current'
     rewrite ^/(.*)/download/stable$                               https://$server_name/$1/download/current permanent;
+    
+    rewrite ^/en/docs/guides/debugging_getting_started/           https://$server_name/en/docs/guides/debugging-getting-started/ permanent;
```
Comment (refack) [inline setup/www/resources/config/nodejs.org]: why just `/en/`?

Comment (fhemberger) [inline setup/www/resources/config/nodejs.org]: You're right. Just saw there's a Korean translation as well.

**Comment by fhemberger** [comment]
@refack Fixed both URLs.

Both renamed files are already online.

File: setup/www/resources/config/nodejs.org (hunk 1 above)
Comment (refack) [inline setup/www/resources/config/nodejs.org]: Wont a regex be better? Or are you concerned only with specific old URLS?

Comment (fhemberger) [inline setup/www/resources/config/nodejs.org]: The guide only exists in those two languages. New translations will use the correct file name. So no regex is needed.

Comment (refack) [inline setup/www/resources/config/nodejs.org]: 👍 You got my LGTM

**Comment by refack** [comment]
LGTM (I'm not write allowed in this repo, so I don't know if it's enough for you)

**Comment by rvagg** [comment]
This is now deployed, but there was a problem on the two lines, @fhemberger I don't quite know what it is but there's some special space character at the begining of both lines, maybe a unicode whitespace of some kind. nginx borked at it and died. Gave us a good chance to exercise our failover solution though and it worked fine!

**Comment by rvagg** [comment]
See https://github.com/nodejs/build/commit/f347666a3a1db760356aa31177ff5521272904a5 FYI, note the weird diff for those two lines.

**Comment by fhemberger** [comment]
Uh, sorry for that! Used the GitHub web editor for this PR … glad it worked out, thanks!

**Pull request merged**